app/lnd/lib/util.js

```diff
@@ -0,0 +1,24 @@
+const fs = require('fs')
+const { promisify } = require('util')
+
+const fsReadFile = promisify(fs.readFile)
+const fsAccess = promisify(fs.access)
+
+module.exports = {
+  validateFileAccess,
+  readFile: fsReadFile
+}
+
+/**
+ * Validates that the file at the specified path can be read
+ * @param {String} path
+ * @returns {Promise<Boolean>}
+ */
+async function validateFileAccess(path) {
```

Comment:
Why `async`? I don't think promise gives us anything here.

Reply:
> Why async?

Well, the method is asynchronous, so it either needs to operate with a callback, a promise, or async. It's probably a case of personal preference but now that async is available natively in node I prefer to use this over promises or callbacks as I just find the code to be cleaner, simpler, and more readable without `.then` and `.catch` all over the place and instead just use good old `try/catch` syntax to achieve the same thing